python/taichi/lang/_texture.py

```diff
@@ -188,8 +181,6 @@ def to_image(self):
             img (PIL.Image.Image): a PIL image in RGB mode, with the same size as source texture.
         """
         assert self.num_dims == 2
-        assert self.dtype == u8
```

Comment:
OOC: do we no longer enforce `dtype` here? If so, the docstring above need to be updated as well. Otherwise we probably still need to assert on `fmt`? ;)